superset/assets/javascripts/explorev2/stores/store.js

```diff
@@ -1629,10 +1629,9 @@ export const fields = {
 };
 
 export function defaultFormData() {
-  const data = {
-    slice_name: null,
-    slice_id: null,
-  };
+  const data = {};
```

Comment:
curious why this:
```
const data = {};
data.slice_name = null;
data.slice_id = null;
```
rather than this:
```
const data = {
  slice_name: null,
  slice_id: null,
};
```